README.md

```diff
@@ -1,2 +1,43 @@
 # brandingUpdate
 
+
+The following table provides a summary of the terms that are changing in accordance with the recent branding guidelines.
+
+When directly referenced in the documentation, all of the terms listed below should be updated to their rebranded counterparts as capabilities of Adobe Experience Platform.
+
+| Original Term  |Updated Term | Explanation  |Previous Example Sentence  | Current Branded Use |
+|---|---|---|---|---|
+|  **Platform Launch Server Side** |  event forwarding | Launch Server Side is now event forwarding. It is written in lower case as it is a capability not a service.  | See the **Launch Server Side** overview and Getting started with Experience **Platform Launch Server Side** for more information.  | See the **event forwarding overview** and Getting started with **event forwarding** for more information.  |
+| **edge configuration**  |  datastream | The term 'edge configuration' should be replaced with the lowercase noun 'datastream'. It is written in lower case as it is not a branded service. | Configure the Web SDK extension for Platform Launch Client Side to send data to the **edge configuration** that you created previously.  | Configure the Web SDK tag extension to send data to the datastream created previously.  |
```

Comment:
Do not use single quotes.

Please simplify these examples, or pick better ones from the docs. There are two different terms that are being replaced in this example, and it's obfuscating what we're trying to demonstrate. We should only be showing where "edge configuration(s)" becomes "datastream(s)"